config/job.js

```diff
@@ -85,6 +85,21 @@ const SCHEMA_ENVIRONMENT = Joi.object()
             }
         }
     });
+const SCHEMA_JOBNAME = Joi.string().regex(Regex.JOB_NAME);
+const SCHEMA_CACHE_VALUE = Joi.string().uri({
+    relativeOnly: true
+});
+const SCHEMA_CACHE_LIST = Joi.array().items(SCHEMA_CACHE_VALUE).min(1);
+const SCHEMA_CACHE_JOB = Joi.object()
+    .pattern(SCHEMA_JOBNAME, SCHEMA_CACHE_LIST)
+    .unknown(false)
+    .length(1);
+const SCHEMA_CACHE_JOBS = Joi.array().items(SCHEMA_CACHE_JOB).min(1);
```

Comment:
Slightly nitpicky, but it may be better to name this schema `SCHEMA_CACHE_JOB_LIST` to keep consistency with the other arrays.